templates/skeleton/app/routes/products.$handle.tsx

```diff
@@ -318,6 +318,14 @@ function AddToCartButton({
           >
             {children}
           </button>
+
+          {fetcher.data?.errors && (
+            <div className="text-red-500">
+              {fetcher.data.errors.map((error: any) => (
+                <div key={error.message}>{error.message}</div>
+              ))}
+            </div>
+          )}
```

Comment:
Thank you for this PR! Could you please post a screenshot of what this looks like rendered?